tests/featureWriters/kernFeatureWriter_test.py

```diff
@@ -45,6 +46,45 @@ class KernFeatureWriterTest(FeatureWriterTest):
 
     FeatureWriter = KernFeatureWriter
 
+    def test_split_pair(self):
+        glyphScripts = {
+            "V": {"Latn"},
+            "W": {"Latn"},
+            "gba-nko": {"Nkoo"},
+            "period": {"Zyyy"},
+            "comma-ar": {"Arab", "Nkoo", "Syrc", "Thaa", "Rohg", "Yezi"},
+            "lam-ar": {"Arab"},
+        }
+
+        # Mixed script pairs don't go anywhere
```

Comment:
Should `ufo2ft` warn in this case?

Reply:
Possibly yes, unless it gets too frequent/noisy. I think we did (still do after this?) issue a warning for mixed horizontal direction pairs (haven't checked since I am now from mobile)

Reply:
This is slightly annoying because the logger object is on the feature writer and the pair is a separate object, so we have to pass the logger in which feels like it breaks encapsulation. Oh well.

Reply:
you don't need to pass the logger, loggers are global, if you need to, simply use `logging.getLogger("name.of.the.logger")`